### PR TITLE
Allow trying all three OCR.space endpoints

### PR DESCRIPTION
**Pull request by itsthejoker**
OCR.space advertises 100% uptime... spread out across all three endpoints. While this is not tremendously kosher, we can play by those rules. This allows the bot to try the next endpoint in line if the first one stops responding for whatever reason.

### CODE REVIEW

File: tor_ocr/main.py [hunk 1]
```diff
@@ -40,8 +41,12 @@
 
 # "helloworld" is a valid API key, however use it sparingly
 __OCR_API_KEY__ = os.getenv('OCR_API_KEY', 'helloworld')
-# __OCR_API_URL__ = 'https://api.ocr.space/parse/image'  # free API url
-__OCR_API_URL__ = 'https://apipro1.ocr.space/parse/image'
+# __OCR_API_URLS__ = ['https://api.ocr.space/parse/image']  # free API url
+__OCR_API_URLS__ = [
+    'https://apipro1.ocr.space/parse/image',  # USA
+    'https://apipro2.ocr.space/parse/image',  # Europe
+    'https://apipro3.ocr.space/parse/image'  # Asia
```
Comment (perryprog) [inline tor_ocr/main.py:48]: Maybe add a space before the comment, so all three comments are aligned?

File: tor_ocr/main.py [hunk 2]
```diff
@@ -130,20 +135,33 @@ def decode_image_from_url(url, overlay=False, api_key=__OCR_API_KEY__):
...
+        except RequestException:
+            # we have a result object here but it's not right.
+            logging.error(
+                f'ERROR {result.status_code} with OCR:\n\nHEADERS:\n '
+                f'{repr(result.headers)}\n\nBODY:\n{repr(result.text)} '
+            )
+
+    if result is None or not result.ok:
+        raise ConnectionError(
+            'Attempted all three OCR.space APIs -- cannot connect!'
```
Comment (perryprog) [inline tor_ocr/main.py:162]: Nit: replace `APIs -- cannot connect!` with `APIs—cannot connect!` (em-dash)

**Review by perryprog — APPROVED** [review]
LGTM, just some nit picks 🙂

**Review by TheLonelyGhost — APPROVED** [review]
Same nits as @perryprog (for the most part), but we're getting quite a few errors logged already so I'd rather call them on the downtime sooner rather than work through code quality nits.

File: tor_ocr/main.py (hunk 1 above)
Comment (itsthejoker) [reply tor_ocr/main.py:48]: I just saw your nits, but official response: PEP-8 calls for two spaces before the # in an inline comment, which is why those are not even. I did think about that, though. :D

File: tor_ocr/main.py (hunk 2 above)
Comment (itsthejoker) [reply tor_ocr/main.py:162]: I'll think about it.

(hint: probably not making this change 😛 )

Comment (perryprog) [reply tor_ocr/main.py:162]: 😂 fair enough—I just really like em-dash 😉

**Pull request merged**